recipes/r-fossil/meta.yaml

```diff
@@ -0,0 +1,91 @@
+{% set version = '0.4.0' %}
+
+{% set posix = 'm2-' if win else '' %}
+{% set native = 'm2w64-' if win else '' %}
+
+package:
+  name: r-fossil
+  version: {{ version|replace("-", "_") }}
+
+source:
+  url:
+    - {{ cran_mirror }}/src/contrib/fossil_{{ version }}.tar.gz
+    - {{ cran_mirror }}/src/contrib/Archive/fossil/fossil_{{ version }}.tar.gz
+  sha256: 37c082fa15ebae89db99d6071b2bb2cad6a97a0405e9b4ef77f62a8f6ad274c1
+
+build:
+  merge_build_host: True  # [win]
+  # If this is a new build for the same version, increment the build number.
+  number: 0
+  # no skip
+
+  # This is required to make R link correctly on Linux.
+  rpaths:
+    - lib/R/lib/
+    - lib/
+
+requirements:
+  build:
+    - {{ posix }}zip               # [win]
+
+  host:
+    - r-base
+    - r-maps
+    - r-shapefiles
+    - r-sp
+
+  run:
+    - r-base
+    - r-maps
+    - r-shapefiles
+    - r-sp
+
+test:
+  commands:
+    # You can put additional test commands to be run here.
+    - $R -e "library('fossil')"           # [not win]
+    - "\"%R%\" -e \"library('fossil')\""  # [win]
+
+  # You can also put a file called run_test.py, run_test.sh, or run_test.bat
+  # in the recipe that will be run at test time.
+
+  # requires:
+    # Put any additional test requirements here.
+
+about:
+  home: http://matthewvavrek.com/programs-and-code/fossil/
+  license: GPL-2
```

Comment:
```suggestion
  license: GPL-2.0-or-later
```

Reply:
I tried this before, but this makes the recipe fail in macOS test, I don't know why....